src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,31 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    /**
+     * In your implementations - create 2-3 models, add them to the list and return
+     *
+     * @return - the list of models
+     */
+    @Override
+    public List<Machine> get() {
```

Comment:
be attentive with task description (4. - https://github.com/mate-academy/jv-pecs)